### ISSUE TITLE
Ignore .git and files in .gitignore

### ISSUE THREAD

**Issue by sebastianquek** [cs2103t/collate#35]

**Comment by sebastianquek** [comment]
Ignoring files in `.gitignore` will add more complexity to the code and is not the main focus of Collate.

**Comment by damithc** [comment]
Yes, I agree. If at all, we can introduce a switch to explicitly exclude files. e.g. 'run for this folder but ignore these files'
